app/models/item.rb

```diff
@@ -7,4 +7,9 @@ class Item < ApplicationRecord
   validates  :quantity, numericality: { greater_than_or_equal_to: 0 }
   validates  :price, numericality: { greater_than_or_equal_to: 0}
 
+  def recalculate_current_quantity 
+    x = line_items.pluck(:quantity).sum
```

Comment:
use descriptive names
```suggestion
    current_quantity = line_items.sum(:quantity)
```

Reply:
changed x-> item_quantity